js/ui/gantt/ui.gantt.dialogs.js

```diff
@@ -7,6 +7,7 @@ export class GanttDialog {
 
         this.infoMap = {};
```

Comment:
this.infoMap = {
  TaskEdit: TaskEditDialogInfo,
  Resources: ResourcesEditDialogInfo
};